Use python3 for installed executable python scripts.

Fedora is a Python3-only distribution:
https://fedoraproject.org/wiki/FinalizingFedoraSwitchtoPython3

This fixes build failures where builders may strictly enforce only
python3 during a transitional phase.

Author: Carlos O'Donell <carlos@redhat.com>

diff --git a/benchtests/scripts/compare_bench.py b/benchtests/scripts/compare_bench.py
index d4370d723f313a0e..2bbb91d6ea9f6c18 100755
--- a/benchtests/scripts/compare_bench.py
+++ b/benchtests/scripts/compare_bench.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 # Copyright (C) 2015-2024 Free Software Foundation, Inc.
 # This file is part of the GNU C Library.
 #
diff --git a/benchtests/scripts/import_bench.py b/benchtests/scripts/import_bench.py
index fac2945f6d36db13..ac40878934490380 100644
--- a/benchtests/scripts/import_bench.py
+++ b/benchtests/scripts/import_bench.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 # Copyright (C) 2015-2024 Free Software Foundation, Inc.
 # This file is part of the GNU C Library.
 #
